### PR TITLE
API enhancements

### PR DESCRIPTION
**Pull request by UladBohdan**
* 401 -> 403 http code error for API
* Author pre-initialization on `npm run init-db` - authors are generated from `users.json`, referred in articles with email then
* `User` model is extended with `imageUrl` field.
* Authors are added to our init data (and also to our high-dev init data on Google Drive)
* Articles now have `imageUrl` for folder image and `videoId` for YouTube video linking fields.
* Authors aka Users are localized: for every User/Author returned `firstName`, `lastName` and `bio` fields are maps locale:String. In addition, `firstName` is required now.

### CODE REVIEW

**Comment by UladBohdan** [comment]
[Author localization!](https://trello.com/c/fMikqxUK/148-author-localization)

**Comment by UladBohdan** [comment]
[add Authors to Article samples](https://trello.com/c/0CcNObzT/96-add-authors-to-article-samples)

**Comment by UladBohdan** [comment]
[User model: add image (we mostly need it for Author)](https://trello.com/c/U8fbaCAC/141-user-model-add-image-we-mostly-need-it-for-author)

**Comment by UladBohdan** [comment]
[to improve author initialization in init-db script: author must sometimes be shared.](https://trello.com/c/AZ0qKZXO/149-to-improve-author-initialization-in-init-db-script-author-must-sometimes-be-shared)

**Comment by UladBohdan** [comment]
[Folder images for Articles](https://trello.com/c/xnKdKdGI/153-folder-images-for-articles)

**Comment by UladBohdan** [comment]
[add picture URL-s and youtube video ID-s to Article model.](https://trello.com/c/Ij9KzQEA/154-add-picture-url-s-and-youtube-video-id-s-to-article-model)

File: src/api/article/article.model.js [hunk 1]
```diff
@@ -43,6 +43,9 @@ const ArticleSchema = new Schema({
     type: Boolean,
     default: true,
   },
+  pictureURL: String,
```
Comment (Drapegnik) [inline src/api/article/article.model.js]: mb `imageUrl`?

File: src/api/article/article.model.js [hunk 2]
```diff
@@ -43,6 +43,9 @@ const ArticleSchema = new Schema({
...
+  // This is a YouTube video ID. Ignored unless Article type is video.
+  videoID: String,
```
Comment (Drapegnik) [inline src/api/article/article.model.js]: imho better use camelCase - `videoId`

File: src/api/article/article.model.js (hunk 1 above)
Comment (UladBohdan) [reply src/api/article/article.model.js]: probably, not sure.

File: src/api/article/article.model.js (hunk 2 above)
Comment (UladBohdan) [reply src/api/article/article.model.js]: debatable. In many case it's ok to use uppercase for such things like ID or URL

File: src/db/articles.json [hunk 3]
```diff
@@ -33,7 +35,9 @@
         "text": "Hello! :)",
         "slug": "slug1-en"
       }
-    }
+    },
+    "authorEmail": "curie@wir.by",
+    "videoID": "SPauxWv1Rnk"
```
Comment (Drapegnik) [inline src/db/articles.json]: you forgot fix it here

Comment (UladBohdan) [reply src/db/articles.json]: fixed.

**Pull request merged**